src/components/Header/index.js

```diff
@@ -23,6 +36,12 @@ export default function Header() {
   const [username] = useState(uid);
   const [anchorEl, setAnchorEl] = useState(null);
   const open = Boolean(anchorEl);
+  const [onHover, setOnHover] = useState({
+    home: false,
+    projects: false,
+    people: false,
+    administrators: false,
+  });
```

Comment:
This is too complex to add styles on hover. You're using Button from material-ui. So you can use this https://smartdevpreneur.com/add-hover-styling-to-a-material-ui-button-with-styled-components/